docs/dm/assets.md

```diff
@@ -37,46 +37,57 @@ Then, open the folder where you have stored your files.
 Last, drag the asset onto the game board. 
 Your asset will now be able to be manipulated.
 
-## Manipulating Assets
+## Manipulating Objects
```

Comment:
Not sure wheter this could be simply a link to the upcoming `gameboard.md` file I'm currently working on. But maybe I will simply merge this into my work when this is done. :)

Reply:
For this, once you have completed that document, I'd recommend replacing this section with a link to that file.